### DISCUSSION TITLE
Mention KnobGallery

### DISCUSSION THREAD

**Discussion by noisecode3** [DISTRHO/DPF#400] [Ideas]
Hi, I think it could be nice to mention [KnobGallery](https://www.g200kg.com/en/webknobman/gallery.php) somewhere in the examples or some documents or just here. I know this is a bit trivial. But when making opensource plugins we are not allowed to use any GUI art that someone have claimed any kind of copying right to or right to not use freely. Its basically almost all you can find on google. Only Public domain, Creative Commons and similar licensed or home made. It took some time for me to find it, maybe it can help others. You can make you're own and modify them also with KnobMan, there is a web version and a download version. Are there more gallery's with vector graphics or supper high resolution out there for opensource?

**Comment by falkTX** [comment]
isnt know gallery pixmap based? we should move away from those into scalable designs

**Comment by noisecode3** [comment]
They are scalable but output is pixel based png yes from the program, its a bit weird I can set the output to be 512, then I have to remember what zoom all the layers was set at, and change it back... I can output them to any size. You can make the GUI scale but if you want resize the window by mouse dragging, you have to make widgets that can read the format or you translate them to widgets some other way. You can make a zoom in the GUI, you can set it to 100% 150% 200% if you want.
